### PR TITLE
Replace mobile background image by a color

### PR DESCRIPTION
**Pull request by ghost**
I've taken the color from the image wit GIMP. It looks more or less the same than before:

screen shot 2015-09-20 at 17 29 41

### CODE REVIEW

**Comment by Flaburgan** [comment]
Why would you want to do that?
And it doesn't look the same on pages where the content doesn't reach the bottom of the screen, like the publisher page.

**Comment by AugierLe42e** [comment]
> Why would you want to do that?

Because the background image slowers content loading on slow networks (particularly in suburbs). I though that, with discussion on #6244, nobody would oppose removing useless content loading...

**Comment by Flaburgan** [comment]
That was the funny part ^^ if you want to load 3MB pictures on mobile, you should not see any problem to use a 22kb pict as a background :)

**Comment by AugierLe42e** [comment]
I want to load what's useful for. Images in streams are useful. Image background that can be replaced by a CSS rule doesn't really look useful ;)

**Comment by svbergerem** [comment]
I guess this is the last open issue to close #4705.

**Comment by AugierLe42e** [comment]
Feel free to merge, then ;)

File: app/assets/stylesheets/colors.scss [hunk 1]
```diff
@@ -22,6 +22,7 @@ $light-green: lighten($green,20%);
 $red: #A80000;
 $blue: #3F8FBA;
 $dark-blue: darken(#0984C8,10%);
+$mobile-background: #E0DFDB;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/colors.scss]: Color `#E0DFDB` should be written as `#e0dfdb`

**Comment by svbergerem** [comment]
I think I'd even prefer the sidebars background that we also use for the desktop view. @Flaburgan what do you think?

**Comment by AugierLe42e** [comment]
No problem for me. It adds consistency.

**Comment by Flaburgan** [comment]
The lesser colors we have, the happier I am.

**Comment by AugierLe42e** [comment]
Ok, here how it looks with new color:

screen shot 2015-10-02 at 10 28 27

File: app/assets/stylesheets/mobile/mobile.scss [hunk 2]
```diff
@@ -26,11 +26,7 @@ code {
   }
 
 body {
-  background: {
-    image: image-url("mobile/hatched-light.jpg");
-    /* scale background image down for iOS retina display */
-    size: 200px;
-  }
+  background-color: $sidebars-sub-background;
```
Comment (svbergerem) [inline app/assets/stylesheets/mobile/mobile.scss]: I was thinking about `$sidebars-background` which we also use for the desktop view as the background for the stream.

Comment (AugierLe42e) [inline app/assets/stylesheets/mobile/mobile.scss]: I tested it. I found it a bit light. I think mobile view needs more contrast than desktop, don't you?

**Comment by AugierLe42e** [comment]
Hmm... I don't know. Maybe you're right, @svbergerem:

**Comment by svbergerem** [comment]
Thank you!

**Pull request merged**